docker/development/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "3.8"
+services:
+  dev_dca:
+    build:
+      context: ../../
+      dockerfile: docker/development/Dockerfile
+    container_name: dca_php
+    volumes:
+    - ./.env:/app/.env
```

Comment:
In de container kun je beter environment vars gebruiken:

```yaml
services:
  dca:
    env_file: ./.env
```

Reply:
Die property kende ik nog niet thanks :)